.editorconfig

```diff
@@ -12,3 +11,6 @@ trim_trailing_whitespace = true
 
 [*.bat]
 end_of_line = crlf
+
+[composer.json]
+indent_size = 2
```

Comment:
This must be removed, we use 4 space indentation for all json files.

Reply:
Removed ident_size property